eclipse-che/che-dashboard
# feat(che-theia): Resolve plug-ins for che-theia editor

### What does this PR do?
Resolve che-theia plug-ins
It embeds the che-theia and devfile library to update devWorkspace templates if there are some che-theia plug-ins
It also resolve optional files content

https://user-images.githubusercontent.com/436777/121185653-9364ac80-c866-11eb-8c60-813f4fc74657.mp4


### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/19927

### how to test PR 
1. Use sample from get started page

OR

2. Use $CHE_HOST/#https://github.com/che-samples/java-spring-petclinic/blob/devfilev2/devfile.yaml as factory link and make sure redhat/java plugin is present after workspace is started

#### Release Notes


#### Docs PR
